test/archivespec.js

```diff
@@ -176,7 +176,7 @@ describe("archive", function () {
           sinon.assert.calledWith(write, sinon.match(helper.uploadParamMatcher("target_format", "zip")));
         });
       });
-      it.skip('should create archive with "zip" format and include multiple resource types', function () {
+      it('should create archive with "zip" format and include multiple resource types', function () {
```

Comment:
This test still fails in all envs

Reply:
Worked locally, I've noticed a few more failed on travis, will review.